src/index.ts

```diff
@@ -44,6 +53,11 @@ export async function run(): Promise<void> {
     if (tagsList.length > 1) {
         await cli.tag(image, tagsList);
     }
+
+    // Restore if original file is changed
+    core.debug("Restoring the original file to avoid any breaking change for future jobs");
+    restoreFile(storageFilePaths[0]);
```

Comment:
wrap this in a try/catch with an error log, but the action should not fail if this fails